.rubocop.yml

```diff
@@ -85,6 +85,10 @@ Metrics/BlockLength:
     - '**/*.rake'
     - 'test/**/*.rb'
 
+Metrics/MethodLength:
+  Exclude:
+    - 'test/**/*.rb'
```

Comment:
Can you update `'test/**/*.rb'` to `'test/rubocop/cop/minitest/global_expectations_test.rb'` and move that change to the .rubocop_todo.yml?